.github/README.md

```diff
@@ -41,7 +41,7 @@ install this version of Perl locally, run the following commands:
 ```
 
 This will configure and compile perl for your platform, run the regression
-tests, and install perl in a subdirectory "localperl" of your home directory.
```

Comment:
I'm not sure if I'm nit-picking here - but I wondered whether we still need the "" quotes when the text is formatted in `code font`. The implication of the font is the same as the implication of the quotes - that this word is a literal token of some form.

I also wondered whether generated HTML page would read more cleanly if you remove the "" around filenames that are now links - ie

does this:

> you should read the detailed instructions in the [INSTALL](https://github.com/rwp0/perl5/blob/blead/INSTALL) file that came with this distribution

look better than this?

> you should read the detailed instructions in the "[INSTALL](https://github.com/rwp0/perl5/blob/blead/INSTALL)" file that came with this distribution

Reply:
Oops, I didn't manage to grab the next line. Your change was

```
 This will configure and compile perl for your platform, run the regression
-tests, and install perl in a subdirectory "localperl" of your home directory.
+tests, and install perl in a subdirectory "`localperl`" of your home directory.
```

Also, oops, github "helpfully" let me review an old commit, but I think that this is still the same change in the current commit.

Reply:
I also think so. Quotes are redundant in Markdown.